public/app/plugins/panel/geomap/style/markers.ts

```diff
@@ -248,7 +247,7 @@ async function prepareSVG(url: string, size?: number): Promise<string> {
       return res.text();
     })
     .then((text) => {
-      text = DOMPurify.sanitize(text, { USE_PROFILES: { svg: true, svgFilters: true } });
+      text = textUtil.sanitizeSVGContent(text);
```

Comment:
I think better if we keep all DOMPurify in one file

Reply:
Since we use `DOMPurify.setConfig` I wonder how this will behave, see: https://github.com/cure53/DOMPurify#persistent-configuration

Will the policy defined by `DOMPurify.setConfig()` also be used in SVG-sanitation? I did some testing and did not get that behavior which is what we want.